Update building-a-contact-form.md (#19570)
gatsbyjs/gatsby: docs/docs/building-a-contact-form.md

## Form submission options in Gatsby

### Getform

> Getform is a form backend platform which offers a free-plan for handling form submissions on static sites. In order to begin receiving form submissions from your Gatsby site, you must first create a form on the platform directing the http POST method to the Getform, by placing the `name` attributes for the fields you want to make visible. (name, email, message etc.)

Reviewer: ```suggestion
Getform is a form backend platform which offers a free-plan for handling form submissions on static sites. In order to begin receiving form submissions from your Gatsby site, you must first create a form on the platform. When creating the form, direct the http POST method to the Getform, by placing the `name` attributes for the fields you want to make visible. (name, email, message etc.)
```
Which platform? Gatsby or Getform?

Author: Getform is the platform which can be used as a form submission handling option for Gatsby.

Reviewer: Right. Just want to add some language to clarify this piece `you must first create a form on the platform`. It isn't clear which platform is being discussed.

```jsx:title=src/pages/contact.js
<form method="post" action="https://getform.io/{your-unique-getform-endpoint}">
  ...
  <label>
    Email
    <input type="email" name="email" />
  </label>
  <label>
    Name
    <input type="text" name="name" />
  </label>
  <label>
    Message
    <input type="text" name="message" />
  </label>
  ...
</form>
```

> Once you've made the changes, you can head over to your contact page and start receiving submissions to the form. You can add multiple email addresses to receive email notifications for the forms created, as well as manipulate the data using Zapier and Webhooks options that are offered. 

Reviewer: We still need to be clearer here as to where you're making the submissions and where you can see them.

## Form submission options in Gatsby

### Getform

> Getform is a form backend platform which offers also a free-plan for handling form submissions on static sites. In order to begin receiving form submissions from your Gatsby site, you must first create a form on the platform directing the http POST method to the Getform, by placing the `name` attributes for the fields you want to make visible. (name, email, message etc.)

Reviewer: ```suggestion
Getform is a form backend platform which offers a free-plan for handling form submissions on static sites. In order to begin receiving form submissions from your Gatsby site, you must first create a form on the platform directing the http POST method to the Getform, by placing the `name` attributes for the fields you want to make visible. (name, email, message etc.)
```

> Getform is a form backend platform which offers a free-plan for handling form submissions on static sites. In order to begin receiving form submissions from your Gatsby site, you must first create a form on the Gatsby site. When creating the form, direct the http POST method to the Getform, by placing the `name` attributes for the fields you want to make visible. (name, email, message etc.)

Reviewer: ```suggestion
Getform is a form backend platform which offers a free-plan for handling form submissions on static sites. Begin by creating a form on your Gatsby site that you can receive submissions from. When creating the form, direct the Http POST method to the Getform, by placing the `name` attributes for the fields you want to make visible. (name, email, message etc.)
```

```jsx:title=src/pages/contact.js
<form method="post" action="https://getform.io/{your-unique-getform-endpoint}">
  ...
  <label>
    Email
    <input type="email" name="email" />
  </label>
  <label>
    Name
    <input type="text" name="name" />
  </label>
  <label>
    Message
    <input type="text" name="message" />
  </label>
  ...
</form>
```

> Once you've made the changes on your Gatsby contact form, you can head over to your contact page and start receiving submissions to the form, the submissions will then be visible on Getform dashboard. You can add multiple email addresses to receive email notifications for the forms created, as well as manipulate the data you see on Getform using Zapier and Webhooks options that are offered. 

Reviewer: ```suggestion
Once you've made the code changes in your contact form, you can head over to the contact page on your site and start submitting data to the form. The submissions will then be visible on the Getform dashboard. You can add multiple email addresses to receive email notifications for the forms created, as well as manipulate the data you see on Getform using Zapier and Webhooks options that are offered. 
```
There are a few different levels here. The form code in Gatsby, the form itself in your site, and the Getform dashboard. I made some recommendations that I hope will clarify which piece we're referring to.

> Once you've made the changes, you can head over to your contact page and then start receiving submissions to the form. You can add multiple email addresses to receive email notifications for the forms created, as well as manipulate the data using Zapier and Webhooks options that are offered. You can find more info on the registration process and form setup [on their website](https://getform.io/) and find code examples (AJAX, reCAPTCHA etc) on their [Codepen](https://codepen.io/getform).

Reviewer 2: ```suggestion
Once you've made the changes, you can head over to your contact page and then start receiving submissions to the form. You can add multiple email addresses to receive email notifications for the forms created, as well as manipulate the data using Zapier and Webhooks options that are offered. You can find more info on the registration process and form setup [on the Getform website](https://getform.io/) and find code examples (AJAX, reCAPTCHA etc) on their [Codepen](https://codepen.io/getform).
```

> Once you've made the changes, you can head over to your contact page and then start receiving submissions to the form. You can add multiple email addresses to receive email notifications for the forms created, as well as manipulate the data using Zapier and Webhooks options that are offered. You can find more info on the registration process and form setup on the [Getform website](https://getform.io/) and find code examples (AJAX, reCAPTCHA etc) on their [Codepen](https://codepen.io/getform).

Reviewer: ```suggestion
Once you've made the changes, you can head over to your contact page and start receiving submissions to the form. You can add multiple email addresses to receive email notifications for the forms created, as well as manipulate the data using Zapier and Webhooks options that are offered. 

You can find more info on the registration process and form setup on the [Getform website](https://getform.io/) and find code examples (AJAX, reCAPTCHA etc) on their [Codepen](https://codepen.io/getform).
```

Reviewer: "you can head over to your contact page and start receiving submissions to the form" can you clarify this a bit? Is there a contact page now in getform? Or is it still in their gatsby site and you're prompting them to send information through the form first?

Author: 2nd one it is! It's still in the gatsby site and you're prompting them to send information through the form.

Reviewer: Cool, can we clarify that in the text?
